app/scripts/controllers/transactions/index.js

```diff
@@ -2430,7 +2432,7 @@ export default class TransactionController extends EventEmitter {
 
     /** The transaction status property is not considered sensitive and is now included in the non-anonymous event */
     let properties = {
-      chain_id: chainId,
+      chain_id: caipChainId, // what to do here?
```

Comment:
probably have left a comment elsewhere, but are we okay with using CAIP chain ID values for the `chain_id` metrics property?